offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/MaintainProductsView.groovy

```diff
@@ -122,16 +128,21 @@ class MaintainProductsView extends FormLayout {
     }
 
     private void setupPanel(){
+        HorizontalLayout formlayout = new HorizontalLayout()
+        formlayout.setSizeFull()
+        this.addComponent(formlayout)
+
         productDescription = new Panel("Product Description")
+        formlayout.addComponents(productDescription)
```

Comment:
```suggestion
        descriptionLayount.addComponents(productDescription)
```

Reply:
If you fix the typo then I will accept the suggestion

Reply:
I think I fixed it 😄